controllers/telemetry/suite_test.go

```diff
@@ -153,7 +153,7 @@ var _ = BeforeSuite(func() {
 
 	metricPipelineReconciler := NewMetricPipelineReconciler(
 		client,
-		metricpipeline.NewReconciler(client, testMetricPipelineReconcilerConfig, &kubernetes.DeploymentProber{Client: client}, overridesHandler))
+		metricpipeline.NewReconciler(client, testMetricPipelineReconcilerConfig, &kubernetes.DeploymentProber{Client: client}, &kubernetes.DaemonSetProber{Client: client}, overridesHandler))
```

Comment:
The line is getting too long, can we maybe break It into multiple lines?

Reply:
done